WellTask_Frontend/src/components/auth/Login.tsx

```diff
@@ -0,0 +1,69 @@
+import { SidePanel } from "@/components/base-component/SidePanel";
+import {
+  FormPanel,
+  type FormField,
+} from "@/components/base-component/FormPanel";
+
+export default function Login() {
+  const panelObj = {
+    title: "Welcome Back!",
+    subtitle: "Get back to your dashboard",
+    showAppButtons: true,
+    position: "right" as "left" | "right",
+    isVisible: true,
+  };
+
+  const formObj: {
```

Comment:
The postfix `obj` is unnecessary. We can remove it.
```suggestion
  const form: {
```